pkg/api/server/swagger.go

```diff
@@ -131,3 +131,13 @@ type swagListContainers struct {
 		//handlers.Container
 	}
 }
+
+// To be determined
+// swagger:response tbd
+type swagTBD struct {
+}
+
+// Filters baude
```

Comment:
Can the comments in this file be improved a bit? I have no idea what's going on here.

Reply:
no they cannot, they are only used by go-swagger to generate documentation.  the one you pick on in particular is just a test. lemme remove it.